suricata/update/rule.py

```diff
@@ -146,6 +146,8 @@ def __str__(self):
         return self.format()
 
     def format(self):
+        if self.noalert and not "noalert;" in self.raw:
+            self.raw = re.sub(r'\)$', " flowbits:noalert;)", self.raw)
```

Comment:
by convention rules always end with sid+rev, would it be possible to insert before the sid?

Reply:
I agree. I will update it in a new branch and pull-request and close this one.